outbound/mx_lookup.js

```diff
@@ -35,7 +35,10 @@ exports.lookup_mx = function lookup_mx (domain, cb) {
         }
         else if (addresses && addresses.length) {
             for (let i=0,l=addresses.length; i < l; i++) {
-                if (obc.cfg.local_mx_ok || !net_utils.is_local_ip(addresses[i].exchange)) {
+                if (
+                    obc.cfg.local_mx_ok ||
+                    await net_utils.is_local_host(addresses[i].exchange).catch(() => null) === false
```

Comment:
The `catch` is just to be extra safe and ignore a potential DNS resolution failure.